web-ui/src/main/resources/catalog/js/edit/EditorController.js

```diff
@@ -538,11 +538,13 @@
         if ($location.search()['justcreated']) {
           // Remove newly created record
           var md = gnCurrentEdit.metadata;
+          var encodedTitle = encodeURI(md.title);
+          var encodedDefaultTitle = encodeURI(md.defaultTitle);
           gnMetadataActions.deleteMd(md).
               then(function(data) {
                 $rootScope.$broadcast('StatusUpdated', {
-                  title: $translate.instant('metadataRemoved',
-                  {title: md.title || md.defaultTitle}),
+                  title: decodeURI($translate.instant('metadataRemoved',
+                  {title: encodedTitle || encodedDefaultTitle})),
```

Comment:
These changes don't seem to resolve the root cause of the problem.  There are most likely other locations that have the same issues.

I found the following which seems to discuss the issue.
https://stackoverflow.com/questions/33121843/angular-translate-sanitize-escape

For geonetwork, it seem like this is set in the following.

https://github.com/geonetwork/core-geonetwork/blob/63ff3ea4ca01d44745fed4ce0d540f7979809046/web-ui/src/main/resources/catalog/js/GnLocale.js#L144-L146

Note sure why 'escape' was previously removed.  

Looks like if we use the following then it should fix the issue.
`$translateProvider.useSanitizeValueStrategy( 'escape', 'sanitizeParameters');`

But not sure if it will introduce other issues?

Reply:
I will run a localhost test see if it can help to relief this issue. As for whoever made the change, I guess we can check Git history and add that person into this conversation.